src/audio/base_fw.c

```diff
@@ -220,6 +220,42 @@ static int basefw_mem_state_info(uint32_t *data_offset, char *data)
 	return 0;
 }
 
+static int basefw_get_dsp_properties(uint32_t *data_offset, char *data)
```

Comment:
@abonislawski so is this what should be used instead of FW_READY in IPC4 to get the fw information after fw boot is successful?

Reply:
No, its a reply for ipc large_config_get **after** fw ready

Reply:
@abonislawski what I meant was in IPC3, we get all these properties along with the FW_READY message. In IPC4, is the expectation that the driver should send a get_large_config() after FW_READY to get these?